editoast/src/views/search.rs

```diff
@@ -654,14 +654,18 @@ pub(super) struct SearchResultItemStudy {
 #[derive(Search, Serialize, ToSchema)]
 #[search(
     table = "search_scenario",
+    migration(src_table = "scenario"),
     joins = "
         INNER JOIN scenario ON scenario.id = search_scenario.id
         INNER JOIN infra ON infra.id = scenario.infra_id",
-    column(name = "id", data_type = "integer"),
-    column(name = "name", data_type = "string"),
-    column(name = "description", data_type = "string"),
-    column(name = "tags", data_type = "string"),
-    column(name = "study_id", data_type = "integer")
+    column(name = "name", data_type = "TEXT", sql = "scenario.name"),
+    column(name = "description", data_type = "TEXT", sql = "scenario.description"),
```

Comment:
```suggestion
    column(name = "name", data_type = "TEXT", sql = "scenario.name", textual_search),
    column(name = "description", data_type = "TEXT", sql = "scenario.description", textual_search),
```

Signals that the column should be indexed to allow [full text search](https://www.postgresql.org/docs/current/textsearch.html) and more, including accent processing. Otherwise you can't find a scenario named "mon scénario" using "scé".